packages/schematics/src/command-line/affected.ts

```diff
@@ -95,6 +92,23 @@ function e2e(apps: string[], rest: string[]) {
   }
 }
 
+function filterNxSpecificArgs(args: string[]): string[] {
+  const nxSpecificFlags = [
+    '--parallel',
+    '--no-parallel',
+    '--base',
+    '--head',
+    '--files',
+    '--uncommitted',
+    '--untracked',
+    '--exclude'
+  ];
+
+  return args.filter(
```

Comment:
You should just leverage yargs to do this sort of filtering (it already has a place where it puts flags which did not match the current command). Arguments without a corresponding flag show up in the `argv._ array` https://github.com/yargs/yargs/blob/master/docs/api.md#argv

Reply:
Unfortunately... they only put arguments in here.. The flags get parsed to their best ability.. I wonder if there's a strict mode?